docs/gtm-interchain.md

```diff
@@ -0,0 +1,73 @@
+
+Leading up to Stargate, IBC Development was led by Chris Goes (Interchain Berlin), while Gaia did not have a maintainer. Go-to-market for Stargate was led by Zaki (Iqlusion). Moving forward, we are aiming to organize Cosmos Hub leadership under Shahan Khatchadourian (Interchain Berlin) and go-to-market led by Jelena Djuric (Informal Systems). Both Shahan and Jelena are being onboarded into these roles with the support of previous leads. 
+
+Execution and completion will be tracked via a meta-project dashboard ([Gaia Roadmap](https://github.com/cosmos/gaia/projects/6)) that incorporates the high level objectives for each phase roll out. Each phase (i.e. Phase 1 - Accounts UX, Phase 2 - Interchain Interchange, etc) will have their own project dashboard that will link to the repos, issues, PRs, etc, where development is taking place.
+
+&quot;_Never doubt that a small group of thoughtful, committed, citizens can change the world. Indeed, it is the only thing that ever has.&quot; - Margaret Mead_
+
+**What&#39;s Needed to Successfully Go to Market?**
+
+1. **Project management**
+
+In sum, project management will focus on surfacing issues, dependencies, relative roadmaps, and coordinating across all stakeholders. Questions that will be considered throughout each phase of the Gaia roadmap:
+
+- What is missing from an integration points perspective with end users?
+
+- Which relationships are missing?
+- What&#39;s the highest priority? Least priority?
+
+2. **Market validation**
+
+The user is paramount. For Gaia and IBC, we see the users being:
+
+  - Atom holders
+  - Existing zones
+  - New zones
+  - Validators on hub and zones
+  - Nodes on hub and zones
+  - Users of hub and zones
+
+We will also want to be tracking outreach to the relevant users per phase to ensure that Hub utility (and IBC where appropriate) are being validated. Spotlights on experiments will continue to validate the utility of the Hub and IBC (i.e. bridge to Ethereum, AMM, etc). We will also be conducting continous outbound outreach to look for partners that are aligned with the interchain mission.
+
+3. **Communications**
+
+Communications should focus on the features of IBC and the Hub that demonstrate the most utility for users. Stories should answer questions such as:
+
+- How does IBC relate to ATOM and the Hub?
+- How does IBC derive its value from the Hub?
+- Why should you build your blockchain with IBC?
+- How does IBC help blockchains become sovereign?
+
+4. **Technical Support**
```

Comment:
I feel like this category could be a more general "Technical Onboarding" and should include things like

- where do developers go to get started building
- how do people find out about whats going on in the interchain, browse zones and interchain activity
- etc.

So not just about "technical support" for building but also for exploring and using the interchain